### default supported ROS message types:
# sensor_msgs/Imu IMU数据
# geometry_msgs/Twist 几何信息
# std_msgs/String 字符串
# (modify/add in ros_sub_pub.hpp if you need more)如果需要更多，请修改/添加 ros_sub_pub.hpp
### max number of send_topics supported: 50 支持的最大发送主题数：50

### list all <IPname: IP> here (IPname can be set arbitrarily) ###
IP:
  self: '*'   # '*' stands for all self IPs
  PC2: 192.168.10.102
  airfly: 192.168.10.108

####### Send these ROS messages to remote robots #######i
## if no send_topics needed, comment all these out
send_topics:
# 无人机消息
# 八叉树点云话题
- topic_name: /octomap_full
  msg_type: octomap_msgs/Octomap
  srcIP: airfly
  srcPort: 3040
# 带点云双目图像话题
- topic_name: /vins_fusion/image_track
  msg_type: sensor_msgs/Image
  max_freq: 50 
  srcIP: airfly 
  srcPort: 3041 
# 历史视觉点云话题(地图)
- topic_name: /vins_fusion/margin_cloud
  msg_type: sensor_msgs/PointCloud
  max_freq: 50
  srcIP: airfly 
  srcPort: 3042 

## 笔记本传来的消息
## 1.启动指令//
recv_topics:
# - topic_name: /PC2/chatter_recv 
#   msg_type: std_msgs/String 
#   srcIP: PC2 
#   srcPort: 3020 


